.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
I use a "langer-than-standard-VGA-resolution" screen, but I only have 92 columns on my terminals/editors. I'm willing to up it to 90 columns, and re-format all of the files I touched to take advantage of it.